### ISSUE TITLE
amazon cant find price, new element #priceblock_ourprice 

### ISSUE THREAD

**Issue by teebu** [dylants/price-finder#112]
Url: https://www.amazon.com/HP-Black-Original-Cartridge-CH561WN/dp/B003H2GBM4/ref=sr_ph_1?ie=UTF8&qid=1479934942&sr=sr-1&keywords=ink+cartridge

It's inside
```
 '#priceblock_ourprice_row #priceblock_ourprice' 
```

**Comment by teebu** [comment]
I tried loading it in cognito, https://www.amazon.com/dp/B003H2GBM4 and there is no price.

**Comment by dylants** [comment]
Yeah, this is interesting. It looks like Amazon won't show their price for this product (and others like it). Reading the explanation shows:

> Why don't we show the price?
> Because our price on this item is lower than the manufacturer's suggested retail price, the manufacturer does not allow us to show you our price until you place the item in your shopping cart. Retailers like Amazon have the legal right to set their own prices independently. Adding the item to your cart allows Amazon to show you our price consistent with our goal of always offering you the lowest possible prices on the widest selection of products.

It doesn't appear that price-finder will work for a product with this restriction :frowning_face: 

For this specific product, if you'd rather use one of the "Other Sellers on Amazon" product listing, I would recommend using that URL directly. So for example, the top one in the current listing as of this writing is this url:
https://www.amazon.com/dp/B003H2GBM4?m=A1DRRDNQTMI7K6

Running it through price-finder, it outputs the following for price, name, and category:

```
16.95
HP 61 Black Original Ink Cartridge (CH561WN)
Other
```

Hope that helps!

**Comment by teebu** [comment]
I'd rather not redirect or use another url, because that involves another request and finding the url as well. 

I think I like my solution, but it may need better CSS? not sure.

**Comment by teebu** [comment]
The way I am using price finder is ASIN directly, not custom urls:

```
function getProductInfoPriceFinder(ASIN, callback){
  const uri = `http://www.amazon.com/dp/${ASIN}`;
  priceFinder.findItemPrice(uri, function(err, price) {
    //console.log('priceFinder price:', price); // 8.91
    return callback(err,price)
  });
}
```

**Comment by dylants** [comment]
Hopefully fixed this issue in #114, please let me know if it does not work after that change. Thanks!
